package.json

```diff
@@ -7,21 +7,50 @@
     "test": "test"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "babel src -x .ts -d lib",
```

Comment:
add `--sourceMap` flag so if error occures when using this lib we will know where in typescript files it occured

Reply:
thats awesome thanks!